src/upload.php

```diff
@@ -17,23 +17,29 @@
 
         $first_run = true;
         $files_exist_counter = 0;
-
+        
+        if ($config['upload_to_root']) {
+            $target_dir = '/';
+        }else {
+            $target_dir = '/'. $config['directory_for_uploads'] . '/';
```

Comment:
Perhaps here we have a check to see if that directory exists, if it doesn't spit out an error?